.gitlab-ci.yml

```diff
@@ -1,23 +0,0 @@
-image: registry.gitlab.com/sorare/crypto/ci:latest
-
```

Comment:
Shouldn't we be creating corresponding github workflows ?

Reply:
Yes, or go with CircleCI; something open-source compliant.

Reply:
Done @smaarn https://github.com/sorare/crypto/actions/runs/1608577756